src/radical/pilot/configs/resource_xsede.json

```diff
@@ -357,12 +357,9 @@
         "task_launch_method"          : "SPARK",
         "mpi_launch_method"           : "MPIRUN_RSH",
         "pre_bootstrap_1"             : [ 
-                                        "module purge",
-                                        "module load TACC",
-                                        "module load python/2.7.12",
+                                        "module load python/2.7.9",
                                         "export TACC_DELETE_FILES=TRUE",
```

Comment:
Hi Andre,

it is the same configuration with wrangler. ( no spark)

login1.wrangler(5)$ module load python/2.7.12
Lmod has detected the following error:  These module(s) exist but cannot be loaded as requested:
"python/2.7.12"

   Try: "module spider python/2.7.12" to see how to load the module(s).
python/2.7.12 doesn't work